hack/tekton/run_create_product_sbom.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Utility testing script designed to mimick the process the
+# create-product-sbom-ta Tekton task uses. Uses the real stage S3 bucket
+# for now.
+#
+# Requirements:
+#   - TPA is running locally on port 8080.
+#   - AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY env vars are set with the
+#       credentials to the stage bucket
+#   - A "snapshot.json" file exists in the data_dir. The snapshot spec should
+#       then point to an image with SBOMs to be augmented.
+#   - A "data.json" merged release data file exists in the data_dir.
+set -eux
+
+data_dir="."
+snapshot_spec="snapshot.json"
+release_data="data.json"
+atlas_api_url="http://localhost:8080"
+retry_s3_bucket="mpp-e1-preprod-sbom-29093454-2ea7-4fd0-b4cf-dc69a7529ee0"
+sbom_path="product-sbom"
```

Comment:
Is it okay to name the sbom file just `product-sbom`, instead of a unique name? IIRC in the old script, a unique name with the product name and version had to be used.